src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx

```diff
@@ -44,7 +44,7 @@ export class ArtworkDetailsAdditionalInfo extends React.Component<
         title: "Condition",
         value:
           canRequestLotConditionsReport &&
-          getENV("ENABLE_REQUEST_CONDITION_REPORT") ? (
+          sd.ENABLE_REQUEST_CONDITION_REPORT ? (
```

Comment:
@yuki24 - The problem is `sd` only renders client-side, so the SSR pass may be compromised. Any chance an update can be made to `getENV` instead?